docs/06-concepts/02-models.md

```diff
@@ -250,6 +250,20 @@ uuidDefaultRandom: UuidValue, default=random
 uuidDefaultUuid: UuidValue, default='550e8400-e29b-41d4-a716-446655440000'
 ```
 
+### Duration
+
+#### Supported Default Values
+
+| Type                          | Keyword  | Description |
+|-------------------------------|----------|-------------|
```

Comment:
Suggestion: Please describe what the "keys" mean.

That "d" is day and "min" is minutes etc.

Reply:
Done!